spec/features/main_chatroom_spec.rb

```diff
@@ -20,33 +15,38 @@
   end
 
   scenario "user sees last five messages when entering the main chat room", js: true do
+    login_with_soundcloud
+    user = User.last
+
     messages = []
     5.times do |n|
-      message = Message.create(body: "message number #{n}",
+      message = Message.create!(body: "message number #{n}",
                                channel: "main",
                                user_id: user.id
                               )
       messages.push(message)
     end
 
-    login_as(user)
+
+
+    visit main_chat_path
 
     messages.each do |message|
-      within(".messages") { expect(page).to have_content(message.body) }
+      within(".messages.main") { expect(page).to have_content(message.body) }
     end
   end
 
-  scenario "User One can see a message posted by User two", js: true do
-    user2 = User.create(name: "Richard",
-                        email: "richard@example.com",
-                        password: "password"
-                       )
-    login_as(user)
-    fill_in("message[body]", with: "message stuff")
-    click_link_or_button("Chat")
+  #scenario "User One can see a message posted by User two", js: true do
+    #user2 = User.create(name: "Richard",
+                        #email: "richard@example.com",
+                        #password: "password"
```

Comment:
Incorrect indentation detected (column 24 instead of 23).<br>Missing space after #.